AppDashboard/lib/app_dashboard_data.py

```diff
@@ -170,6 +172,75 @@ def __init__(self, helper=None):
     """
     self.helper = helper or AppDashboardHelper()
 
+  def build_dict(self):
+    LOOKUP_DICT = {}
+    LOOKUP_DICT['cloud_stats'] = {"title":"Cloud Status",
```

Comment:
I think this function would be more readable if the keys were included in the dict literal definition:
```
LOOKUP_DICT = {
  'cloud_status': {'title': 'Cloud Status', ...},
  'database_stats': {'title: 'Database', ...},
  ...
}
```